modules/virt-cloning-a-datavolume.adoc

```diff
@@ -30,7 +30,7 @@ spec:
   source:
     pvc:
       namespace: "<source-namespace>" <2>
-      name: "<my-favorite-vm-disk>" <3>)
+      name: "<my-favorite-vm-disk>" <3>
```

Comment:
@bgaydosrh Can you please ack that this update to a virt topic is okay?

Reply:
Ack. Thanks for fixing this @bergerhoffer.